.changeset/afraid-pumas-enjoy.md

```diff
@@ -0,0 +1,10 @@
+---
+"@kaizen/date-picker": patch
+---
+
+* Deprecate `FilterDateRangePicker`
```

Comment:
Include a note for where to find the replacement. I think this is also a good place to note the replacement's API changes given we haven't got the flexibility on the `components/FilterDRP` docs to add migration notes yet.